src/components/Modal/Modal.css

```diff
@@ -0,0 +1,97 @@
+
+.modal__wrapper {
+  font-family: Aktiv Grotesk, -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif;
+  position: absolute;
+  z-index: 1000;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+}
+
+.modal__wrapper.--hidden {
+  display: none;
+}
+
+.modal__overlay {
+  background: rgba(0, 0, 0, 0.6);
+  width: 100%;
+  height: 100%;
+  position: fixed;
+}
+
+.modal {
+  position: fixed;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  z-index: 2;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  max-width: 700px;
```

Comment:
Lets use REM values instead, so that the UI is easily scaled according the given root font-size. With this there will be less trouble adapting the UI to match different screen sizes.

Remember that REM multiplies its value by the root font-size. So if the the root font-size is 16px, to achieve 700px the value should be 43.75rem (700/16 = 43.75).

The same principle should be followed everywhere you are working with pixels. The exceptions are when you want something to be of very low values, 1px/2px.